crates/rolldown/src/ast_scanner/impl_visit.rs

```diff
@@ -90,6 +90,40 @@ impl<'me, 'ast: 'me> Visit<'ast> for AstScanner<'me, 'ast> {
     walk::walk_member_expression(self, expr);
   }
 
+  fn visit_for_of_statement(&mut self, it: &ast::ForOfStatement<'ast>) {
+    if it.r#await {
+      if let Some(format) = self.options.as_ref().map(|option| &option.format) {
+        if !format.keep_esm_import_export() {
```

Comment:
I kind of fail to understand what `keep_esm_import_export` means when reading esbuild's code.

Reply:
my bad, it should change to `keep_esm_import_export_syntax`,